testdata/dnn/download_models.py

```diff
@@ -816,3 +822,9 @@ def save(self, r):
     for m in models:
         print(m)
         m.get()
```

Comment:
BTW, we can slightly simplify final verification:
```.python
def get(self):
    ...
    return self.verify()
...
failedSums = []
for m in models:
        print(m)
        if not m.get():
            failedSums += m.filename
if failedSums:
    print('Checksum verification failed for following files: {}'.format(failedSums))
    exit(1)
```

Reply:
@mshabunin could you add necessary changes please.

Reply:
@sturkmen72 , I've updated my branch, you can pull it and verify in your environment.

Reply:
@mshabunin thank you. i tested and believe it is ready!